.github/workflows/ci-tests.yaml

```diff
@@ -18,7 +18,7 @@ jobs:
       uses: actions/setup-go@93397bea11091df50f3d7e59dc26a7711a8bcfbe # v4.1.0
       with:
         # renovate: datasource=golang-version depName=go
-        go-version: 1.20.5
```

Comment:
we ned to fix renovate to stick on go minor version 1.20.x

Reply:
-> https://github.com/cilium/proxy/pull/336

Reply:
restrict the go version range seems to work -> go update is removed (-> stick on 1.20.x)